core/docker/Dockerfile.importer

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+
+# Install prerequisites
+RUN apt-get update && apt-get install -y curl
+
+RUN apk add --no-cache curl
```

Comment:
@rcrichton I think its useful for us to have control of the layers within this image so we can install more modules as needed. This could be something like a MySQL client for importing directly to a database.

Initially we also tried some of the images that come pre-installed with curl but couldn't get them working nicely with the bash script.

One question i have is whether each service should have its own importer server or if we should bundle them as one `component` config import

Reply:
Yeah ok makes sense. We might have to publish this Dockerfile somewhere then as people may want to run this without having the source code.

I think maybe a single importer script would work for now. We could split it up if that seems necessary.

Reply:
@rcrichton for sure, we can def make this a Docker image which can be publicly downloaded and re-used. I would say this should then belong on the openhie docker repo
https://hub.docker.com/u/openhie